docs/ui/components/Tag/StatusTag.tsx

```diff
@@ -20,10 +20,7 @@ export const StatusTag = ({ status, type, note }: StatusTagProps) => {
       )}
       css={[tagStyle, type === 'toc' && tagToCStyle]}>
       {status === 'experimental' && <Stars02Icon className="icon-xs text-palette-pink11" />}
-      <span css={labelStyle}>
-        {formatName(status)}
-        {note && `: ${note}`}
-      </span>
+      <span css={labelStyle}>{status ? formatName(status) + (note ? `: ${note}` : '') : note}</span>
```

Comment:
This was required to use the `<StatusTag />` with just `notes` (as in, for just adding "SDK XX+")

<img width="931" alt="image" src="https://github.com/expo/expo/assets/1203991/18f4345a-f485-401a-b648-e9f5edee2992">